src/main/java/com/team4/testingsystem/converters/notifications/TestExpiredNotificationConverter.java

```diff
@@ -0,0 +1,21 @@
+package com.team4.testingsystem.converters.notifications;
+
+import com.team4.testingsystem.dto.NotificationDTO;
+import com.team4.testingsystem.entities.Notification;
+import com.team4.testingsystem.enums.NotificationType;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TestExpiredNotificationConverter extends SingleNotificationConverter {
+    @Override
+    public NotificationDTO convertToDTO(Notification notification) {
+        return notificationBuilder(notification)
+                .level(notification.getTest().getLevel().getName())
+                .build();
+    }
+
+    @Override
+    public NotificationType converterType() {
+        return NotificationType.TEST_EXPIRED;
+    }
```

Comment:
I think, we can pretty easy make the same with level

Reply:
What means "the same"?
Is it about pushing `level` up to the abstract class?

Reply:
Yes

Reply:
But `TestDeassignedNotificationConverter` doesn't set the level